### ISSUE TITLE
[1.8.1-beta20] Modules not found

### ISSUE THREAD

**Issue by pmogollons** [meteor/meteor#10494]
Ive been having some issues with latest beta of 1.8.1 where I get a ```Cannot find module``` error. I think it might be related to the latest @zodern improvements to the build process (thanks by the way).

The error goes away by updating the file where im importing the module, then restarting meteor.

Ill try to create a reproduction of the issue and post it here.

```js
Uncaught (in promise) Error: Cannot find module './TasksComments'
    at makeMissingError (modules-runtime.js?hash=d3c3e5d67c95f97a60888bda7373292efad3be5e:232)
    at Module.require (modules-runtime.js?hash=d3c3e5d67c95f97a60888bda7373292efad3be5e:251)
    at Module.moduleLink [as link] (modules.js?hash=b149bdfc524f17b87638d490a25cdcf142f5c580:327)
    at eval (/imports/ui/condos/components/TasksModal.js:135)
    at dynamic-import.js?hash=2fa5396ca744d8857eb51b6ab65b873369440dad:138
    at fileEvaluate (modules-runtime.js?hash=d3c3e5d67c95f97a60888bda7373292efad3be5e:346)
    at Module.require (modules-runtime.js?hash=d3c3e5d67c95f97a60888bda7373292efad3be5e:248)
    at Module.moduleLink [as link] (modules.js?hash=b149bdfc524f17b87638d490a25cdcf142f5c580:327)
    at eval (Tasks.js:1)
    at dynamic-import.js?hash=2fa5396ca744d8857eb51b6ab65b873369440dad:138
    at fileEvaluate (modules-runtime.js?hash=d3c3e5d67c95f97a60888bda7373292efad3be5e:346)
    at Module.require (modules-runtime.js?hash=d3c3e5d67c95f97a60888bda7373292efad3be5e:248)
    at Module.moduleLink [as link] (modules.js?hash=b149bdfc524f17b87638d490a25cdcf142f5c580:327)
    at getNamespace (dynamic-import.js?hash=2fa5396ca744d8857eb51b6ab65b873369440dad:187)
    at dynamic-import.js?hash=2fa5396ca744d8857eb51b6ab65b873369440dad:40
    at meteor.js?hash=857dafb4b9dff17e29ed8498a22ea5b1a3d6b41d:1234
```

**Comment by benjamn** [comment]
Good find! Any chance of a small reproduction?

**Comment by jamesmillerburgess** [comment]
We have also seen this error and it was related to the case sensitivity of a folder name - not sure if it is the same thing.

**Comment by pmogollons** [comment]
I have been trying to reproduce it for some time already with no success in a new --react project. I saw this error almost 10 days ago and rolled back to 1.8.0.2 yesterday we went again to 1.8.1 beta and it happened once.

I dont think its the same cause. I just change the file extension on the import then restarted meteor and it worked.

```import TaskComments from './TaskComments';```
To
```import TaskComments from './TaskComments.js```

Then restarted and it worked again even if I changed back to ```./TaskComments``` with no extension.

Ill keep trying to reproduce it in the empty project, but if it happens in our project im not sure how can I share that. 

If its a cache issue ill need to zip the entire project and send it as the repro?

**Comment by pgiani** [comment]
I see the same problem, I need to change a file, stop meteor and start again for it to see the changes,

**Comment by arggh** [comment]
I was being blind and did not see this issue, creating my own at https://github.com/meteor/meteor/issues/10496. I believe these are the same issue. #10496 now has a reproduction that I believe demonstrates this problem in a fairly minimal setup.

**Comment by pmogollons** [comment]
@arggh thanks for the repro, I too belive this might be the same issue. Im wondering why I was not able to repro the issue on a clean react project. Not even in my project happened that consistently. But good that your repro got @benjamn on the right path to fix it.

**Comment by benjamn** [comment]
This should be fixed with `meteor update --release 1.8.1-rc.0` (just published)!
